CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+## Contribution Guidelines
+
+**Note:** If these contribution guidelines are not followed your issue or PR might be closed, so
+please read these instructions carefully.
+
+_See also: [Flutter's code of conduct](https://flutter.dev/design-principles/#code-of-conduct)_
```

Comment:
This is Flutter's code of conduct. Not sure if invertase has their own or if we should include Flutter's code of conduct.

cc @Salakar